default.nix

```diff
@@ -80,6 +80,12 @@ with rec
               )
             );
 
+          patchedSources =
+            lib.optionals (builtins.hasAttr "patch" toplevelCargotoml)
+              (map (po: src + "/" + po.path)
+                (lib.collect (as: lib.isAttrs as && builtins.hasAttr "path" as)
```

Comment:
Do you know what else than `crates.io` can be patched?  Is it safe to just assume that we'll "vendor" all patches that have a `path`?

Reply:
I think it's safe.